ovos_PHAL_plugin_homeassistant/logic/connector.py

```diff
@@ -296,85 +335,101 @@ def event_listener(self, message):
                 self.event_listeners[entity_id](message)
 
     @staticmethod
-    def _device_entry_compat(devices: dict, enable_debug):
+    def _device_entry_compat(devices: dict):
         disabled_devices = list()
         for idx, dev in devices.items():
-            if dev.get('disabled_by'):
-                if(enable_debug):
-                    LOG.debug(f'Ignoring {dev.get("entity_id")} disabled by '
-                            f'{dev.get("disabled_by")}')
+            if dev.get("disabled_by"):
```

Comment:
Consider inverting this to log added devices instead of disabled; I get a lot of disabled entities I didn't even know existed, i.e. for a fire7 I use as a controller:
```
2023-05-26 17:24:21.346 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_current_version disabled by integration
2023-05-26 17:24:21.350 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_app_rx_gb disabled by integration
2023-05-26 17:24:21.354 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_app_tx_gb disabled by integration
2023-05-26 17:24:21.362 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_app_memory disabled by integration
2023-05-26 17:24:21.378 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring binary_sensor.fire7_app_inactive disabled by integration
2023-05-26 17:24:21.382 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_app_importance disabled by integration
2023-05-26 17:24:21.385 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_ringer_mode disabled by integration
2023-05-26 17:24:21.392 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_audio_mode disabled by integration
2023-05-26 17:24:21.398 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring binary_sensor.fire7_headphones disabled by integration
2023-05-26 17:24:21.419 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring binary_sensor.fire7_mic_muted disabled by integration
2023-05-26 17:24:21.422 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring binary_sensor.fire7_speakerphone disabled by integration
2023-05-26 17:24:21.424 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring binary_sensor.fire7_music_active disabled by integration
2023-05-26 17:24:21.428 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_volume_level_alarm disabled by integration
2023-05-26 17:24:21.436 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_volume_level_call disabled by integration
2023-05-26 17:24:21.440 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_volume_level_music disabled by integration
2023-05-26 17:24:21.458 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_volume_level_ringer disabled by integration
2023-05-26 17:24:21.465 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_volume_level_notification disabled by integration
2023-05-26 17:24:21.467 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_volume_level_system disabled by integration
2023-05-26 17:24:21.471 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_volume_level_dtmf disabled by integration
2023-05-26 17:24:21.479 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring binary_sensor.fire7_is_charging disabled by integration
2023-05-26 17:24:21.494 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_battery_health disabled by integration
2023-05-26 17:24:21.500 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_battery_temperature disabled by integration
2023-05-26 17:24:21.503 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_battery_power disabled by integration
2023-05-26 17:24:21.510 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_bluetooth_connection disabled by integration
2023-05-26 17:24:21.513 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring binary_sensor.fire7_bluetooth_state disabled by integration
2023-05-26 17:24:21.517 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_ble_transmitter disabled by integration
2023-05-26 17:24:21.525 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_beacon_monitor disabled by integration
2023-05-26 17:24:21.542 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_screen_brightness disabled by integration
2023-05-26 17:24:21.544 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_screen_off_timeout disabled by integration
2023-05-26 17:24:21.546 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_do_not_disturb_sensor disabled by integration
2023-05-26 17:24:21.551 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring binary_sensor.fire7_work_profile disabled by integration
2023-05-26 17:24:21.553 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring binary_sensor.fire7_device_locked disabled by integration
2023-05-26 17:24:21.556 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring binary_sensor.fire7_device_secure disabled by integration
2023-05-26 17:24:21.558 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring binary_sensor.fire7_keyguard_locked disabled by integration
2023-05-26 17:24:21.561 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring binary_sensor.fire7_keyguard_secure disabled by integration
2023-05-26 17:24:21.576 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_last_used_app disabled by integration
2023-05-26 17:24:21.581 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_last_reboot disabled by integration
2023-05-26 17:24:21.583 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_last_update_trigger disabled by integration
2023-05-26 17:24:21.585 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_light_sensor disabled by integration
2023-05-26 17:24:21.588 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_wifi_connection disabled by integration
2023-05-26 17:24:21.595 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_wifi_bssid disabled by integration
2023-05-26 17:24:21.597 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_wifi_ip_address disabled by integration
2023-05-26 17:24:21.610 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_wifi_link_speed disabled by integration
2023-05-26 17:24:21.615 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring binary_sensor.fire7_wifi_state disabled by integration
2023-05-26 17:24:21.617 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_wifi_frequency disabled by integration
2023-05-26 17:24:21.618 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_wifi_signal_strength disabled by integration
2023-05-26 17:24:21.620 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_public_ip_address disabled by integration
2023-05-26 17:24:21.625 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_network_type disabled by integration
2023-05-26 17:24:21.629 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_next_alarm disabled by integration
2023-05-26 17:24:21.632 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_last_notification disabled by integration
2023-05-26 17:24:21.635 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_last_removed_notification disabled by integration
2023-05-26 17:24:21.637 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_active_notification_count disabled by integration
2023-05-26 17:24:21.657 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_media_session disabled by integration
2023-05-26 17:24:21.659 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring binary_sensor.fire7_interactive disabled by integration
2023-05-26 17:24:21.661 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring binary_sensor.fire7_doze_mode disabled by integration
2023-05-26 17:24:21.663 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring binary_sensor.fire7_power_save disabled by integration
2023-05-26 17:24:21.665 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_internal_storage disabled by integration
2023-05-26 17:24:21.667 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_external_storage disabled by integration
2023-05-26 17:24:21.671 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_current_time_zone disabled by integration
2023-05-26 17:24:21.673 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_total_rx_gb disabled by integration
2023-05-26 17:24:21.675 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_total_tx_gb disabled by integration
2023-05-26 17:24:21.678 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_app_standby_bucket disabled by integration
2023-05-26 17:24:21.695 - enclosure - ovos_PHAL_plugin_homeassistant.logic.connector:_device_entry_compat:343 - DEBUG - Ignoring sensor.fire7_volume_level_accessibility disabled by integration
2023-05-26 17:24:21.745 - enclosure - ovos_PHAL_plugin_homeassistant:build_devices:214 - DEBUG - Device added: binary_sensor.updater - binary_sensor - None
2023-05-26 17:24:21.747 - enclosure - ovos_PHAL_plugin_homeassistant:build_devices:214 - DEBUG - Device added: switch.harmony_hub_hdmi_1 - switch - None
2023-05-26 17:24:21.749 - enclosure - ovos_PHAL_plugin_homeassistant:build_devices:214 - DEBUG - Device added: switch.harmony_hub_hdmi_2 - switch - None

```

Reply:
Nevermind; I see added devices are logged elsewhere